pyccel/parser/syntactic.py

```diff
@@ -441,23 +442,9 @@ def _visit_NameNode(self, stmt):
             return Symbol(val)
 
     def _visit_ImportNode(self, stmt):
-        if not isinstance(stmt.parent, (RedBaron, DefNode)):
-            errors.report(PYCCEL_RESTRICTION_IMPORT,
-                          bounding_box=stmt.absolute_bounding_box,
-                          severity='error')
-
-        if isinstance(stmt.parent, DefNode):
-            errors.report(PYCCEL_RESTRICTION_IMPORT_IN_DEF,
-                          bounding_box=stmt.absolute_bounding_box,
-                          severity='error')
-
-        # in an import statement, we can have seperate target by commas
-        ls = self._visit(stmt.value)
-        ls = get_default_path(ls)
-        expr = Import(ls)
-        expr.set_fst(stmt)
-        self.insert_import(expr)
-        return expr
+        errors.report(PYCCEL_UNEXPECTED_IMPORT,
+                      bounding_box=stmt.absolute_bounding_box,
+                      severity='error')
```

Comment:
can you also add this part :
ls = self._visit(stmt.value)
ls = get_default_path(ls)
expr = Import(ls)
expr.set_fst(stmt)
self.insert_import(expr)
return expr
because we continue converting even if we have an error